app/server.rb

```diff
@@ -0,0 +1,90 @@
+require 'sinatra'
+require 'data_mapper'
+require 'rack-flash'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, "postgres://localhost/5hitter_#{env}")
+
+require './app/models/sheet'
+require './app/models/user'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
+enable :sessions
+set :session_secret, 'super secret'
+use Rack::Flash
+use Rack::MethodOverride
+
+set :public_folder, Proc.new { File.join(root, '..', "public") }
+
+get '/' do
+  @sheets = Sheet.all#(:order => :created_at.desc)
+  erb :index
+end
+
+get '/users/new' do
+  @user = User.new
+  erb :'users/new'
+end
+
+post '/users' do
+  @user = User.create(:name => params[:name],
```

Comment:
adjusting params could be extracted to a separate method to reduce the size of this one